### ISSUE TITLE
Bullets are Slow

### ISSUE THREAD

**Issue by Kiyahitayika** [ScorpioStation/ScorpioStation#323]
**Issue Description**:

- Bullets/Projectiles are slow
- James says so (been saying so for a while)
- Zarak says so (they're slower than floor tiles when you throw floor tiles. Zarak removes floor tiles to look for their ant-friends)
- TheMaveran says you can dodge tasers with rest (also, rest should have a cooldown, but that's a separate issue).

**What did you expect to happen**:

- projectiles to be faster, I guess

**What happened instead**:

- Projectiles are slower than the Great Boston Molasses Flood of 1919. Only one person died in that. And not from a projectile.

**Why is this bad/What are the consequences**:

- You can dodge projectiles fairly easily

**Steps to reproduce the problem**:

- Pew pew

**When did the problem start happening**:

- Since Arksoft started providing the guns.

**Extra information**:

**Comment by nok-ko** [comment]
>     * Projectiles are slower than the Great Boston Molasses Flood of 1919. Only one person died in that. And not from a projectile.
The death toll was actually 21 people. But in actually-on-topic news... Yeah. 
> TheMaveran says you can dodge tasers with rest (also, rest should have a cooldown, but that's a separate issue).
Can't you dodge all projectiles with rest?

**Comment by TheMaveran** [comment]
Yeah, but with tasers it’s the most impactful cause low-ish fire rate and they instakill you more or less.

**Comment by Kiyahitayika** [comment]
> The death toll was actually 21 people. But in actually-on-topic news... Yeah.

👀 oh crepes

**Comment by Kiyahitayika** [comment]
https://github.com/ScorpioStation/ScorpioStation/pull/193/files
